accessible/workspaceview.component.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Blockly Demos: AccessibleBlockly
```

Comment:
@NeilFraser -- Why include the 'And all similar files" bit in the title? That doesn't make sense to me.